Workbooks/ConnectedVehicle/PlatformAccount Message Counts/PlatformAccount Message Counts.workbook

```diff
@@ -0,0 +1,169 @@
+{
+  "version": "Notebook/1.0",
+  "items": [
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "parameters": [
+          {
+            "id": "487a9f92-5350-4eec-8d68-15bdf2470550",
+            "version": "KqlParameterItem/1.0",
+            "name": "Resource",
+            "label": "Platform Account",
+            "type": 5,
+            "isRequired": true,
+            "value": "value::1",
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.connectedvehicle/platformaccounts": true
+              },
+              "additionalResourceOptions": [
+                "value::1"
+              ]
+            }
+          }
+        ],
+        "style": "pills",
+        "queryType": 0,
+        "resourceType": "microsoft.connectedvehicle/platformaccounts"
+      },
+      "conditionalVisibility": {
+        "parameterName": "1",
+        "comparison": "isEqualTo",
+        "value": "2"
+      },
+      "name": "resource param"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "<p style=\"font-weight:900\"> Message In/ Message Out Counts </p>\r\nTime Range: 7 Days "
```

Comment:
nit: prefer markdown syntax like `#` to make titles/etc instead of using html and css.

you could also use time range parameters instead of hardcoding to 7 days?

Reply:
Agree on both. I was checking in an existing workbook made by someone (fixing the hardcoded resource reference). I knew the styling needed work, but appreciate the tips here

Reply:
added time range param and moved this to the title property